cocos/ui/UIPageView.cpp

```diff
@@ -409,16 +469,16 @@ void PageView::doLayout()
     _doLayoutDirty = false;
 }
 
-void PageView::movePages(float offset)
+void PageView::movePages(Vec2 offset)
```

Comment:
If the direction is Vertical, shouldn't the page view prevent Horizontal moving action ? Otherwise, it may cause some unwanted effect.